Brainfuck/ocaml/program.yaml

```diff
@@ -0,0 +1,6 @@
+prepare:
+  - ocamlc bf.functional.ml -o bf.functional
+  - ocamlc bf.imperative.ml -o bf.imperative
+run:
+  - ./bf.functional {PROGRAM}
+  - ./bf.imperative {PROGRAM}
```

Comment:
I think the tests right now depend on there being a single `run`.

Reply:
Ahh! I'll handle that in the test script

Reply:
Done, can you have a look?

Reply:
Updated it slightly after a merge conflict but yeah looks ok now